CONTRIBUTING.rst

```diff
@@ -1170,7 +1170,11 @@ To install yarn on macOS:
 
     export PATH="$HOME/.yarn/bin:$PATH"
 
-4.  Install third-party libraries defined in ``package.json`` by running the
+4.  Install third-party libraries defined in ``package.json`` by running the following command
```

Comment:
I don't think this should be the 4th step of `To install yarn on macOS:`, this is not part of the yarn install process.

There is a section bellow to run the server in dev mode, maybe we could add that as the first step before the `yarn run dev` ?
```
    # Starts a web server that manages and updates your assets as you modify them
    # You'll need to run the webserver in debug mode too: ``airflow webserver -d``
    yarn run dev
```

Reply:
Something needs to be added to the doc. Even if I follow all of them, when I run the server and do `yarn run dev` and webserver in debug mode, the UI won't render the react components saying assets not found.